MIDIchlorians/MIDIchlorians/Session.swift

```diff
@@ -36,20 +36,12 @@ class Session: Object {
     }
 
     private func initialisePadGrid() {
-        //for demo
-        let demoSounds = Config.sound
         for page in 0..<numPages {
             pads.append([])
             for row in 0..<numRows {
                 pads[page].append([])
-                for col in 0..<numCols {
+                for _ in 0..<numCols {
```

Comment:
I removed this in my PR too. But if you want to merge this in then I'll jut remove the conflicts